generator/source/faq/validator.rst

```diff
@@ -96,7 +96,7 @@ The sawtooth.validator.transaction_families setting is ignored using dev-mode co
 
 What is the difference between ``sawtooth-validator --peers {list}`` and ``sawtooth-validator --seeds {list}``?
 ---------------------------------------------------------------------------------------------------------------
-There are two peering modes in sawtooth: static and dynamic. The static peering mode requires the ``--peers`` arg to connect to other peer validators. Whereas, in the dynamic peering mode the ``--peers`` if specified will be processed and then use ``--seeds`` for the initial connection to the validator network and to start topology build-out (discovery and connection to more peer validators).
+There are two peering modes in sawtooth: static and dynamic. The peering mode can be specified either using the CLI option or by using the validator config file. The CLI option is ``--peering static`` for the static peering mode and ``--peering dynamic`` for the dynamic peering mode. The static peering mode requires the ``--peers`` arg to connect to other peer validators. Whereas, in the dynamic peering mode the ``--peers`` if specified will be processed and then use ``--seeds`` for the initial connection to the validator network and to start topology build-out (discovery and connection to more peer validators).
```

Comment:
I realize this isn't wrapped 80 chars, but probably should be.  Here are some issues:

- "Whereas, in the dynamic peering mode the ``--peers`` if specified..." -> "When dynamic peering mode is enabled, the ``--peers`` option will be processed as an initial set of peers, and the ``--seeds`` option is used as an initial set of peers, as well as used to start discovery of further peer validators."

Reply:
Yes